src/test_driver/openiotsdk/integration-tests/common/fixtures.py

```diff
@@ -39,75 +67,180 @@ def rootDir():
 
 @pytest.fixture(scope="session")
 def fvp(request):
-    if request.config.getoption('fvp'):
-        return request.config.getoption('fvp')
+    if request.config.getoption("fvp"):
+        return request.config.getoption("fvp")
     else:
-        return shutil.which('FVP_Corstone_SSE-300_Ethos-U55')
+        return shutil.which("FVP_Corstone_SSE-300_Ethos-U55")
 
 
 @pytest.fixture(scope="session")
 def fvpConfig(request, rootDir):
-    if request.config.getoption('fvpConfig'):
-        return request.config.getoption('fvpConfig')
+    if request.config.getoption("fvpConfig"):
+        return request.config.getoption("fvpConfig")
     else:
-        return os.path.join(rootDir, 'config/openiotsdk/fvp/cs300.conf')
+        return os.path.join(rootDir, "config/openiotsdk/fvp/cs300.conf")
 
 
 @pytest.fixture(scope="session")
 def telnetPort(request):
-    return request.config.getoption('telnetPort')
+    return request.config.getoption("telnetPort")
 
 
 @pytest.fixture(scope="session")
 def networkInterface(request):
-    if request.config.getoption('networkInterface'):
-        return request.config.getoption('networkInterface')
+    if request.config.getoption("networkInterface"):
+        return request.config.getoption("networkInterface")
     else:
         return None
 
 
 @pytest.fixture(scope="session")
 def otaProvider(request, rootDir):
-    if request.config.getoption('otaProvider'):
-        return request.config.getoption('otaProvider')
+    if request.config.getoption("otaProvider"):
+        return request.config.getoption("otaProvider")
     else:
-        return os.path.join(rootDir, 'out/chip-ota-provider-app')
+        return os.path.join(rootDir, "out/chip-ota-provider-app")
 
 
 @pytest.fixture(scope="session")
 def softwareVersion(request):
-    if request.config.getoption('softwareVersion'):
-        version = request.config.getoption('softwareVersion')
-        params = version.split(':')
+    if request.config.getoption("softwareVersion"):
+        version = request.config.getoption("softwareVersion")
+        params = version.split(":")
         return (params[0], params[1])
     else:
         return ("1", "0.0.1")
 
 
+@pytest.fixture
+def gdbPlugin(pytestconfig):
+    yield pytestconfig.getoption("--gdbPlugin")
+
+
+@pytest.fixture
+def kvsStore(pytestconfig):
```

Comment:
`kvsStore` was removed. Now only the PS  store is supported by default. I think we can remove this fixture and its dependency

Reply:
this has been simplified to not include the tdb store fixture. I kept the structure. Do you want me to remove everything ?

Reply:
yes, I think the less code to maintain the better

Reply:
Done.